treeherder/perf/models.py

```diff
@@ -108,33 +107,22 @@ class PerformanceDatumManager(models.Manager):
     Convenience functions for operations on groups of performance datums
     """
 
-    # Performance sheriffs check these daily
-    MAIN_REPOS = [
-        'autoland',
-        'mozilla-inbound',
-        'mozilla-beta',
-        'mozilla-central'
-    ]
-
-    def _should_quit(self, started_at, max_overall_runtime):
+    @classmethod
+    def _should_quit(cls, started_at, max_overall_runtime):
         now = datetime.datetime.now()
         elapsed_runtime = now - started_at
 
         return max_overall_runtime < elapsed_runtime
 
-    def cycle_data(self, repository, cycle_interval, chunk_size, sleep_time,
-                   unsheriffed_repos_expire_days, logger,
-                   started_at, max_overall_runtime):
+    def cycle_data(self, cycle_interval, max_chunk_size,
+                   logger, started_at, max_overall_runtime):
         """Delete data older than cycle_interval, splitting the target data
 into chunks of chunk_size size."""
-        logger.warning('Cycling Perfherder data from {0} repository...'
-                       .format(repository.name))
-        max_timestamp = self._determine_max_timestamp(repository.name,
-                                                      cycle_interval,
-                                                      unsheriffed_repos_expire_days)
+        logger.warning('Cycling Perfherder data...')
```

Comment:
I think this would add more noise to papertrail and isn't as useful as the success and exceptions warnings.

Reply:
Agree.